manifests/init.pp

```diff
@@ -88,12 +88,12 @@
   ### START Nginx Configuration ###
   Optional[Enum['on', 'off']] $absolute_redirect             = undef,
   Enum['on', 'off'] $accept_mutex                            = 'on',
-  $accept_mutex_delay                                        = '500ms',
-  $client_body_buffer_size                                   = '128k',
+  String $accept_mutex_delay                                 = '500ms',
+  String $client_body_buffer_size                            = '128k',
   String $client_max_body_size                               = '10m',
-  $client_body_timeout                                       = '60s',
-  $send_timeout                                              = '60s',
-  $lingering_timeout                                         = '5s',
+  String $client_body_timeout                                = '60s',
```

Comment:
I think there's a reasonable chance that people have overridden these timeouts with `Integers` since the `s` for seconds is optional in the config file??

Reply:
@smortex Great spot! :)